chain/x/zoracle/internal/keeper/request.go

```diff
@@ -24,6 +24,51 @@ func (k Keeper) GetRequest(ctx sdk.Context, id int64) (types.Request, sdk.Error)
 	return request, nil
 }
 
+// Request attempts to create new request.
+// An error is returned if some conditions failed
```

Comment:
// Request attempts to create a new request. An error is returned if some conditions failed.